torch/csrc/autograd/function.h

```diff
@@ -112,7 +113,14 @@ struct TORCH_API Function : std::enable_shared_from_this<Function> {
   /// Evaluates the function on the given inputs and returns the result of the
   /// function call.
   variable_list operator()(variable_list&& inputs) {
-    profiler::RecordFunction rec(this);
+    profiler::RecordFunction rec(this, [&inputs]() {
```

Comment:
Use after move!

Reply:
wait, aren't we first using inputs in RecordFunction start callback (by contract), and only then inputs are moved?

Reply:
I'll make sure that lambda is not used in end_callbacks by clearing it in RecordFunction after start callbacks are finished